.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: python
+
+python:
+  - "3.5"
```

Comment:
It's working, that's just the way Travis displays it. See https://travis-ci.org/tomchristie/django-rest-framework?branch=master for an example